libs/ui/autocomplete/package.json

```diff
@@ -37,5 +37,11 @@
     "@terminus/ui-validation-messages": "^1.0.0",
     "rxjs": "^6.5.0",
     "tslib": "^1.10.0"
-  }
+  },
+  "scripts": {
```

Comment:
Do we still need all these scripts in each package?

Reply:
yes we do. This is to define how the schematics of each package gets built.